FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/include/tcp_dump_packets.h

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Could packet dumping please be separated into its own PR? (At least, based on the assumption that it's independent of the mem stats changes, above.)

Reply:
@dcgaws:
I combined the two because they want to change a **common** **file**: `include/IPTraceMacroDefaults.h`.
Also they want to create a new directory `/tools`, which sits next to `source` and `include`.
And a last reason is time: I saved a lot of time by testing them in one go and writing a single PR.